next.config.mjs

```diff
@@ -80,6 +80,7 @@ export default withBundleAnalyzer(
                 {
                     source: '/(.*)',
                     headers: [
+                        // TODO: Enable CSP for production.
```

Comment:
When I started working on the branch, I was facing CSP issues so I temporarily disabled it. I left the comment there to remind myself to enable in the future. I'll remove the comment since the CSP has been fixed.